docs/warnet.md

```diff
@@ -138,16 +149,16 @@ options:
 
 ### `warnet graph import-json`
 Create a cycle graph with nodes imported from lnd `describegraph` JSON file,
-and additionally include 7 extra random outbounds per node. Include lightning
-channels and their policies as well.
-Returns XML file as string with or without --outfile option.
+    and additionally include 7 extra random outbounds per node. Include lightning
```

Comment:
did apidocs.py add this weird whitespace?

Reply:
i suppose it did, im just going to remove the entire command since its not implemented anyway